<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlotPreferencesDialog</class>
 <widget class="QDialog" name="PlotPreferencesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>333</width>
    <height>666</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Plot Preferences</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>590</y>
     <width>161</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>400</y>
     <width>291</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title">
    <string>Exported Plots</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>31</y>
      <width>244</width>
      <height>113</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Resolution (DPI)</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="spinBoxResolution"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Horizontal image size [in]</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="doubleSpinBoxHSize"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Vertical image size [in]</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="doubleSpinBoxVSize"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>291</width>
     <height>341</height>
    </rect>
   </property>
   <property name="title">
    <string>General</string>
   </property>
   <widget class="QSpinBox" name="spinBoxAxesLWidth">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>60</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Plot line width</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Axes font size  [points]</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxFontSize">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>90</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Axes line width </string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBoxLWidth">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>30</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>216</width>
      <height>151</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QCheckBox" name="checkBoxCompactScales">
       <property name="text">
        <string>Use compact scales</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBoxCompactLegend">
       <property name="text">
        <string>Use compact legend</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBoxTitle">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Show Plot Title</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBoxColoredFrame">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Colored Frame </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBoxTimeStamp">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Display Timestamp</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButtonFont">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>120</y>
      <width>84</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PlotPreferencesDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PlotPreferencesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonFont</sender>
   <signal>clicked()</signal>
   <receiver>PlotPreferencesDialog</receiver>
   <slot>selectAxisFont()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>211</x>
     <y>155</y>
    </hint>
    <hint type="destinationlabel">
     <x>148</x>
     <y>332</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>selectAxisFont()</slot>
 </slots>
</ui>
